en/sql/schema/table_stmt.rst

```diff
@@ -1437,6 +1437,18 @@ If you set the type, size, and attribute to apply to a new column with the **CHA
 When you change data types using the **CHANGE** clause or the **MODIFY** clause, the data can be modified. For example, if you shorten the length of a column, the character string may be truncated if the value of configuration parameter alter_table_change_type_strict is set to **no**. But if the parameter value is set to **yes**, the change or modify is not allowed and it returns an error.
 the configuration parameter allow_truncated_string also affect the similar as alter_table_change_type_strict.
 
+When changing the type of the column specified AUTO_INCREMENT, it can't be changed to a column type that not be allowed to used with the AUTO_INCREMENT. For example, if you try to change the column type of "a" from INT to VARCHAR with the ALTER statement as shown below, an error occurs.
```

Comment:
It would be better to say "it cannot be changed to a column type that is not allowed to be used with the AUTO_INCREMENT."